core/platform/transactions/gae_transaction_services.py

```diff
@@ -20,15 +20,39 @@
 
 
 def run_in_transaction(fn, *args, **kwargs):
-    """Run a function in a transaction."""
+    """Run a function in a transaction.
+    Either all of the operations in the transaction are applied,
+    or none of them are applied.
+    If an exception is raised, the transaction is likely not safe to
+    commit, since TransactionOptions.ALLOWED is used.
+
+    Args:
+        fn: A function (or callable) to be called.
+        *args: Variable length argument passed to the callable.
+        **kwargs: Arbitrary keyword arguments passed to the callable.
+
+    Returns:
+      Whatever fn() returns.
+
+    Raises:
+        Whatever fn() raises; datastore_errors.TransactionFailedError
```

Comment:
Follow style elsewhere:

    *: Whatever fn() raises.
    datastore_errors.TransactionFailedError: The transaction fails.